packages/restlessness-core/src/Project/templates/index.ts

```diff
@@ -58,9 +58,8 @@ const generateOfflineServerlessJson = (projectName: string) => `{
 }
 `;
 
-// "service": "${projectName}-${serviceName}",
 const generateServiceServerlessJson = (projectName: string, serviceName: string): string => `{
```

Comment:
This I will move inside JsonServices model, because we don't use at Project create, as we do for offline and shared-resources ones.

Reply:
right👍🏽